index.html

```diff
@@ -6,10 +6,241 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Quiz</title>
+    <link rel="stylesheet" href="https://objective-banach-71e419.netlify.app/main.d8c72a9941bec826e9fd.css">
+    <link rel="stylesheet" href="style.css">
 </head>
 
 <body>
 
+    <nav class="top-navigation-component d-flex flex-align-center bg-white-full">
+        <div class="left d-flex flex-justify-center text-xl">
+            QUIZ
+        </div>
+        <div class="center d-flex flex-center">
+            <input class="navbar-input" type="text" placeholder="search">
+        </div>
+        <div class="right d-flex flex-align-center flex-justify-evenly">
+            <button class="btn btn-primary">Create +</button>
+            <svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none"
+                stroke="#000000" stroke-width="2" stroke-linecap="round" stroke-linejoin="round">
+                <path d="M5.52 19c.64-2.2 1.84-3 3.22-3h6.52c1.38 0 2.58.8 3.22 3" />
+                <circle cx="12" cy="10" r="3" />
+                <circle cx="12" cy="12" r="10" />
+            </svg>
+        </div>
+    </nav>
+
+    <main class="main-content">
+        <div class="banner">
+            <img class="banner-img"
+                src="https://res.cloudinary.com/clouduser/image/upload/v1645300727/download_j6yos8.jpg"
+                alt="solar eclipse quiz">
+            <div class="banner-text">
+                <p class="heading-xl">
+                    Solar eclipses:
+                </p>
+                <p class="heading-xl">When is the next one?</p>
+                <div class="quiz-heading-metrics d-flex flex-justify-evenly">
+                    <p class="text-xl">1237 Questions</p>
+                    <p class="text-xl">615 Answers</p>
+                    <p class="text-xl">100K+ Views</p>
+                </div>
+                <button class="btn btn-primary">Take the Quiz</button>
+            </div>
+        </div>
+
+        <p class="heading-lg">Popular Quiz</p>
```

Comment:
You could probably use a header tag instead of p tag. Do this for other headings as well